ingenialink/eoe/network.py

```diff
@@ -134,10 +199,9 @@ def _build_eoe_command_msg(cmd, node=1, data=None):
         """
         if data is None:
             data = bytes()
-        cmd_field = cmd.encode("utf-8")
-        node_field = f"{node:0{EOE_MSG_NODE_SIZE}d}".encode("utf-8")
-        data_field = data + NULL_TERMINATOR * (EOE_MSG_DATA_SIZE - len(data))
-        return cmd_field + node_field + NULL_TERMINATOR + data_field + NULL_TERMINATOR
+        cmd_field = cmd.to_bytes(EoENetwork.EOE_MSG_CMD_SIZE, "little")
+        data_field = data + EoENetwork.NULL_TERMINATOR * (EoENetwork.EOE_MSG_DATA_SIZE - len(data))
```

Comment:
NULL terminator?

Reply:
It is a null character to fill the data until the 53 bytes. Maybe we can change the name to NULL_CHARACTER, or similar. What do you think?